src/plugins/i18nCountries.js

```diff
@@ -11,6 +11,7 @@ import it from 'i18n-iso-countries/langs/it.json'
 import ja from 'i18n-iso-countries/langs/ja.json'
 import ko from 'i18n-iso-countries/langs/ko.json'
 import pl from 'i18n-iso-countries/langs/pl.json'
+import pt from 'i18n-iso-countries/langs/pt.json'
```

Comment:
@ivangomes Please create a file
`src/plugins/i18nCountries/locales/pt_BR.json`
with contents of [this file](https://github.com/michaelwittig/node-i18n-iso-countries/blob/master/langs/pt.json)
and import it here like this:
`import pt_BR from './i18nCountries/locales/pt_BR.json`